tests/wpt/mozilla/tests/mozilla/window_resource_performance.html

```diff
@@ -0,0 +1,32 @@
+<!doctype html>
+<!-- TODO use test harness  -->
+<html>
+  <head>
+  </head>
+  <body onload="loadResources()">
+    <script>
+       function loadResources()
+       {
+          var image1 = new Image();
+          image1.onload = resourceTiming;
+          image1.src = 'https://www.w3.org/Icons/w3c_main.png';
+       }
+
+       function resourceTiming()
+       {
+           var resourceList = window.performance.getEntriesByType("resource");
+           assert
+           for (i = 0; i < resourceList.length; i++)
+           {
+              if (resourceList[i].initiatorType == "img")
+              {
+                 assert_true(resourceList[i] instanceof PerformanceResourceTiming, "Should be PerformanceResourceTiming");
```

Comment:
This page doesn't actually work correctly, right? Can we remove it?

Reply:
Actually, since it's hard to get a sense of what parts of the API are working from the automated tests, let's turn this page into a straightforward smoketest using testharness.js, just demonstrating the bits that are expected to work correctly. This will make it easier for contributors to test their changes when fixing the various TODOs.

Reply:
i'm actually inclined to remove it--i stuck it when i was figuring out the changes from the old api and the new. The tests in tests/wpt/web-platform-tests/resource-timing/ are thorough and pretty descriptive

resource-timing.html and test-resource-timing are the most useful until we have the timing allow check

Thoughts?